.github/actions/sanity-check/action.yml

```diff
@@ -39,7 +39,7 @@ runs:
         reviewdog_version: v0.10.2
 
     - shell: bash
-      run: ./mvnw -B -q checkstyle:checkstyle-aggregate
+      run: ./mvnw spotless:check
```

Comment:
Remove the next step and review dog submodule, now we don't have checkstyle result to perform review dog.